### DISCUSSION TITLE
Use IMAP instead of SMTP to store emails

### DISCUSSION THREAD

**Discussion by Metabaron1** [simple-login/app#617] [Self-hosting question]
Hi
I'd like your opinion about my problem I'm tring to imagine a solution.
I'm self hosting and I use my ISP mailbox to store my emails. But there is a too much aggressive antispam filter that bounces some emails, whenever I use authenticated or not SMTP relay host. The only way I can imagine to bypass this antispam is using IMAP instead of SMTP...
So do you think that there a way to send the emails using IMAP instead of SMTP? From Postfix? 
Thanks

Edit: but finally I suppose Simplelogin would not be compatible with email tranfered with IMAP ...

**Comment by ageless-mushroom** [comment]
Someone else may be able to provide a more in-depth explanation but IMAP is a protocol designed for mail retrieval, not sending. SMTP is the protocol for sending.

**Reply by Metabaron1** [reply]
@timotree 
Yes, but I was think about something like enabling Postfix storage (or some local maibox for my domain) for email from sl-email module,  and enabling IMAP between this local storage and my ISP mailbox.

But this will break Simplelogin I think

**Reply by nguyenkims** [reply]
I wonder if maybe instead of sending emails with Postfix, upload the email directly into your ISP mailbox can solve this? This can be done by overriding this line https://github.com/simple-login/app/blob/e4ed192cce8fd6b30f96c27f4f5af3ff4ba16639/email_handler.py#L840
